crates/egui_extras/src/loaders.rs

```diff
@@ -104,5 +112,8 @@ mod ehttp_loader;
 #[cfg(feature = "image")]
 mod image_loader;
 
+#[cfg(feature = "image")]
```

Comment:
This is not really what the `image` feature means: https://docs.rs/egui_extras/latest/egui_extras/

Reply:
The description is: `Add support for loading images with the [image](https://docs.rs/image) crate`.
The image_loader is loading DynamicImages from the [image](https://docs.rs/image) crate. Im not quite sure as what I should label it. I created it for the use with gifs, but it can be used with images that were already preprocessed

Reply:
It looks to me like nothing in `raw_image_loader` is using the `image` crate, but maybe I'm missing something